### PR TITLE
Master l10n de new skr csn

### PR DESCRIPTION
**Pull request by csnauwaert**
Task: 31826

### CODE REVIEW

File: addons/l10n_de/__init__.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import models
```
Comment (xmo-odoo) [inline addons/l10n_de/__init__.py]: Incorrect!

File: addons/l10n_de/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+import datev
```
Comment (xmo-odoo) [inline addons/l10n_de/models/__init__.py]: No! (also what kind of name is "datev"?)

File: addons/l10n_de/models/datev.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+    def _get_tax_vals(self, company, tax_template_to_tax):
+    	vals = super(AccountTaxTemplate, self)._get_tax_vals(company, tax_template_to_tax)
+    	vals['datev_correction_code'] = self.datev_correction_code
+    	return vals
+
+class AccountTax(models.Model):
+	_inherit = "account.tax"
+
+	datev_correction_code = fields.Char(size=2, help="2 digits code use by Datev")
```
Comment (qdp-odoo) [inline addons/l10n_de/models/datev.py]: rename into l10n_de_datev_code

File: addons/l10n_de/models/datev.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+class AccountTaxTemplate(models.Model):
+    _inherit = 'account.tax.template'
+
+    datev_correction_code = fields.Char(size=2)
```
Comment (qdp-odoo) [inline addons/l10n_de/models/datev.py]: rename into l10n_de_datev_code

**Comment by csnauwaert** [comment]
Has been merged

**Pull request closed without merging**